src/runtime/hashmap.go

```diff
@@ -154,34 +153,61 @@ func hashmapLenUnsafePointer(m unsafe.Pointer) int {
 	return hashmapLen((*hashmap)(m))
 }
 
+//go:inline
```

Comment:
Is there a straightforward way to check change in binary size by looking at the CI? I checked https://github.com/tinygo-org/tinygo/actions/runs/4527779105/jobs/7974008883?pr=3605 but could not find a diff inside the log.